pkg/services/folder/folderimpl/store.go

```diff
@@ -6,7 +6,6 @@ import (
 	"github.com/grafana/grafana/pkg/services/folder"
 )
 
-// Store is the interface which a folder store must implement.
```

Comment:
this comment is still valid, though it's now `store` instead of `Store`. Can we please leave it in place for the documentation?

Reply:
oops, this was an rebase incident, will add it back